pytition/pytition/settings/base.py

```diff
@@ -275,3 +276,6 @@
 MAINTENANCE_MODE_IGNORE_ADMIN_SITE = True
 # Following setting prevents error 500, due to issue #76 of django_maintenance_mode
 MAINTENANCE_MODE_STATE_FILE_PATH = os.path.join(BASE_DIR, 'maintenance_mode_state.txt')
+
+# Set default region to France in order to accept phone numbers without specifying prefix (+33)
```

Comment:
I would reword it as "Set default region to France with regard to accepting phone numbers without international prefix. The +33 prefix for France is therefore optional, but it might be mandatory to add a prefix for other countries. This does not change anything for translations."
In order to make it crystal clear. Better safe than sorry!